### ISSUE TITLE
"Failed to load Script Extender library"?

### ISSUE THREAD

**Issue by StormCloudzz** [Norbyte/bg3se#332]
I recently re-downloaded Baldur's Gate 3 and updated this mod to the latest version, but I get this error and I'm not 100% sure what it means.

![Screenshot (1232)](https://github.com/Norbyte/bg3se/assets/80630851/8c034a35-1e12-45b6-813c-7e561a861e9b)

**Comment by Norbyte** [comment]
This is usually caused by the VC++ runtime not being up to date .
Try getting this:
https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Comment by Trisolarist** [comment]
I had the same error as you did, totally out of the blue, not after re-installing anything or a new Hotfix release. 
Here are the things I've tried but did not work for me:

a) updated VC++ (as suggested above by Norbyte)
b) re-downloaded the script extender using bg3mm
c) replaced BG3SE AppData Folder with V15 Developer Version (see [https://www.nexusmods.com/baldursgate3/mods/3800?tab=description](url))

Finally, this works:

In bg3mm settings -> preferences -> Updater, change Update channel from "Devel" to "Release". That's it. 

I've only changed my bg3mm settings once since installed and I forgot which one is the default option. I still don't know what went wrong but this just works. Hope this might help if you still have the problem.
